python/orca/src/bigdl/orca/learn/tf2/ray_estimator.py

```diff
@@ -549,6 +549,7 @@ def predict(self,
         from bigdl.orca.data.tf.data import Dataset
 
         if isinstance(data, DataFrame):
+            data.rdd.cache()
```

Comment:
Since the data will already be computed when putting to ray, I think it is better for us to cache it so that when saving we don't need to re-calculate?